packages/environment-check/src/checks/getInstalled.ts

```diff
@@ -33,9 +33,9 @@ async function getExtensionsBAS(): Promise<{ [id: string]: { version: string } }
     const versions = files
         .filter((vsixFile) => isExtensionRequired(vsixFile))
         .reduce((returnObject, current) => {
-            const lastIndex = current.lastIndexOf('-');
-            const id = current.slice(0, lastIndex);
-            const version = current.slice(lastIndex + 1).split('.vsix')[0];
+            const idVersion = current.split('.vsix')[0];
```

Comment:
Hi @cianmSAP, I've noticed that the file name `getInstalled.ts` is camelCase, while all other files are using kebap-case. 
Could you please rename the file to `get-installed.ts`?

Reply:
Updated